CHANGELOG.md

```diff
@@ -1,4 +1,47 @@
 ## 0.6.0 (Unreleased)
+
```

Comment:
Add line before enhancements section:
Recommendation: Before upgrading read the [MongoDB Atlas Provider 0.6.0: Upgrade Guide](https://www.terraform.io/docs/providers/mongodbatlas/guides/0.6.0-upgrade-guide.html)